openai/assistant_event_handler.py

```diff
@@ -143,10 +159,25 @@ def on_end(self):
 
     @override
     def on_exception(self, exception: Exception) -> None:
-        logger.error(f"Exception occurred while executing thread: {exception}")
-
-    def get_thread_messages(self):
-        return self.thread_messages
+        error_message = f"Exception occurred while executing thread: {exception}"
+        logger.error(error_message)
+        self.response.update({'status': 0, 'message': error_message})
```

Comment:
0 status code is used for success, please any +ve or -ve no. like -1

Reply:
Incorporated